.graphqlconfig

```diff
@@ -1,5 +1,5 @@
 {
-  "schemaPath": "libraries/graphql_schemas/schema.graphql",
+  "schemaPath": "schema.graphql",
```

Comment:
I do this because schema was linted by eslint because it was it library folder, but autogenerated schema contain invalid characters (#) and so throwed errors. So I made easiest thing

Are you talking about multiple schemas?